packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js

```diff
@@ -452,7 +448,7 @@ export default compose(
 							rootClientId
 						);
 					} else {
-						replaceBlocks( clientIds, clipboardBlock );
+						replaceBlocks( clientId, clipboardBlock );
```

Comment:
Even though it uses `replaceBlocks` under the hood, shouldn't we use `replaceBlock()` here for semantic accuracy?

Reply:
Good idea, I've updated it in https://github.com/WordPress/gutenberg/pull/46699/commits/69e76ed7b10f080de453ac39f67bd0d4122d8c01